staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/register.go

```diff
@@ -53,12 +54,15 @@ func AddToGroupVersion(scheme *runtime.Scheme, groupVersion schema.GroupVersion)
 		&GetOptions{},
 		&DeleteOptions{},
 	)
-	scheme.AddConversionFuncs(
-		Convert_versioned_Event_to_watch_Event,
-		Convert_versioned_InternalEvent_to_versioned_Event,
-		Convert_watch_Event_to_versioned_Event,
-		Convert_versioned_Event_to_versioned_InternalEvent,
+	scheme_err := scheme.AddConversionFuncs(
+			Convert_versioned_Event_to_watch_Event,
+			Convert_versioned_InternalEvent_to_versioned_Event,
+			Convert_watch_Event_to_versioned_Event,
+			Convert_versioned_Event_to_versioned_InternalEvent,
 	)
+	if scheme_err != nil {
+		fmt.Errorf("scheme.AddConversionFuncs: %v", scheme_err.Error())
```

Comment:
This doesn't actually do anything - fmt.Errorf returns an error object, but then you have to do something with it.  Either AddToGroupVersion should be changed to return an error, or we need to panic or glog.Warningf / glog.Errorf / glog.Fatalf